spec/lib/evss/ppiu/service_spec.rb

```diff
@@ -49,4 +49,61 @@
       end
     end
   end
+
+  describe '#update_payment_information' do
+    let(:request_payload) do
+      {
+        'accountType' => 'Checking',
+        'financialInstitutionName' => 'Fake Bank Name',
+        'accountNumber' => '1234',
+        'financialInstitutionRoutingNumber' => '021000021'
+      }
+    end
+
+    context 'with a valid a valid evss response' do
+      it 'returns a payment information response object' do
```

Comment:
For any `it` blocks containing multiple expectations, perhaps add an `:aggregate_failures`, to ensure that all of the expectations run, should one fail?  i.e.

```ruby
it 'returns a payment information response object', :aggregate_failures do
```